src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -23,26 +22,14 @@ export const GenericSuccess = ({ status, text, errorMessage }) => <Status icon="
 GenericSuccess.propTypes = GenericStatus.propTypes;
 GenericSuccess.defaultProps = GenericStatus.defaultProps;
 
-const ErrorOverlay = ({ errorMessage }) => (
-  <Popover position="right" size="regular" title="Errors">
-    <div>{errorMessage}</div>
-  </Popover>
-);
-
-ErrorOverlay.propTypes = {
-  errorMessage: PropTypes.string,
-};
-
-ErrorOverlay.defaultProps = {
-  errorMessage: null,
-};
-
 // Generic error status component
 // If the errorMessage property isn't empty its contents are
 // shown in a Popover.
 export const GenericError = ({ status, text, errorMessage }) =>
   errorMessage ? (
-    <OverlayStatus icon="error-circle-o" text={text} overlay={<ErrorOverlay errorMessage={errorMessage} />} />
+    <OverlayStatus icon="error-circle-o" header={<div>{text}</div>}>
+      {<div>{errorMessage}</div>}
```

Comment:
```suggestion
      <div>{errorMessage}</div>
```

Reply:
I already fixed it when @suomiy commented about that :-( ... probably forgot to commit, now it's fixed and commited :-)